index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <nav class="navBar">
+      <div>
+        <h1>MyLogo</h1>
+      </div>
+      <button class="navBtn">
+        <i class="fa-solid fa-bars"></i>
+      </button>
+    </nav>
+    <header>
+      <div class="about">
+        <h2>Hey There.<br />I'm Oluwaseyi.</h2>
+        <p>I'm a Software Developer</p>
+        <div class="description">
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+        </div>
+        <div class="iconDiv">
+          <img src="images\Icon -GitHub.png" alt="github icon" id="git"/>
+          <img src="images\LinkedIn.png" alt="linked icon" />
+          <img src="images\Enabled.png" alt="Angelist" />
+          <img src="images\Disabled.png" alt="Twitter logo" />
+          <img src="images\M.png" alt="M icon" />
```

Comment:
- [ ] It is better to use a `ul` tag to group the list of icons, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.
- [ ] Also, please customize this section to have links to your social media account as this is your portfolio and not just a template